cabot/cabotapp/models.py

```diff
@@ -459,7 +459,6 @@ def save(self, *args, **kwargs):
             self.cached_health = ''
             self.calculated_status = Service.CALCULATED_PASSING_STATUS
         ret = super(StatusCheck, self).save(*args, **kwargs)
-        self.update_related_services()
```

Comment:
With many checks running at once, updating services after every check run (and therefore likely multiple at a time) is unnecessary and can overload celery